sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -260,7 +260,7 @@ object FunctionRegistry {
     expression[Pow]("pow", true),
     expression[Pow]("power"),
     expression[ToRadians]("radians"),
-    expression[Rint]("rint"),
+    expression[Rint]("rint", true),
```

Comment:
this too

Reply:
I reference https://github.com/apache/spark/blob/a89006aba03a623960e5c4c6864ca8c899c81db9/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala#L66.
So the default pretty name of `Rint` is 'Round',  I think we can use this method to make it as 'Rint'.